content/en/docs/Tasks/creating-operator-bundle.md

```diff
@@ -50,7 +50,7 @@ Flags:
 For example, to generate the `annotations.yaml` and `Dockerfile` for the example bundle mentioned above, the command for the `generate` task is: 
 
 ```bash
-$ opm alpha bundle generate --directory /etcd --package etcd --channels stable --default stable
+$ opm alpha bundle generate --directory ./etcd --package etcd/manifests --channels stable --default stable
```

Comment:
The input for `--package` is actually correct here. The input is supposed to be the name of the package, I.e `etcd` and not the location of the manifests. 

As for the value for --directory, it does mention this command is in relation to the bundle example mentioned above in the doc, but I see the value of adding the `.` before `/etcd` to be absolutely specific.